src/horus_api.h

```diff
@@ -35,20 +35,75 @@
 #include <stdint.h>
 #include "modem_stats.h"
       
-#define HORUS_MODE_BINARY            0
-#define HORUS_MODE_RTTY              1
+/* Horus API Modes */
+#define HORUS_MODE_BINARY            0  // Legacy binary mode
+#define HORUS_MODE_BINARY_256BIT     1  // New 256-bit LDPC-encoded mode
+#define HORUS_MODE_BINARY_128BIT     2  // New 128-bit LDPC-encoded mode
+#define HORUS_MODE_RTTY              99 // RTTY Decoding
+
+
+// Settings for Legacy Horus Binary Mode (Golay Encoding)
+#define HORUS_BINARY_NUM_BITS                   360   /* fixed number of coded bits */
```

Comment:
Rather than having to explain what protocol #defines refer too with a comment, lets make it clean in the #define name - see comments above re protocol naming.